zap/src/main/java/org/parosproxy/paros/network/HttpHeader.java

```diff
@@ -113,7 +113,7 @@ public abstract class HttpHeader implements java.io.Serializable {
     public static final String LINK = "link";
     public static final String REFRESH = "refresh";
     public static final Pattern patternCRLF = Pattern.compile("\\r\\n", Pattern.MULTILINE);
-    public static final Pattern patternLF = Pattern.compile("\\n", Pattern.MULTILINE);
+    public static final Pattern patternLF = Pattern.compile("(?<!\\r)\\n", Pattern.MULTILINE);
```

Comment:
This is a public pattern better not change its behaviour. Introduce an internal pattern instead.

Reply:
It wasn't used.

Reply:
By whom? It's public anyone can use it.

Reply:
Fair point.